src/freenet/node/PeerNode.java

```diff
@@ -5828,4 +5828,9 @@ private boolean verifyReferenceSignature(SimpleFieldSet fs) throws ReferenceSign
 	protected final byte[] getPubKeyHash() {
 	    return peerECDSAPubKeyHash;
 	}
+
+	// FIXME Move callers into freenet/node/ (or copy the byte[]).
+    public byte[] getPeerECDSAPubKeyHash() {
+        return peerECDSAPubKeyHash;
```

Comment:
I don't see a need to add this as another FIXME, please fix it instead (by copying the `byte[]` and adding Javadoc, that is). It's not used frequently anyways.